ci/version-check.ts

```diff
@@ -13,9 +15,8 @@ const validateVersion = async () => {
   if (response.status >= 400) {
     return Promise.reject(`Invalid response for tag request ${response.status}`);
   }
-  const data = await response.json();
-  const releases = data.map(({ name }: { name: string }) => name);
-  console.log('Existing releases', releases);
+  const data = (await response.json()) as VersionData[];
+  const releases = (data as []).map(({ name }: { name: string }) => name);
```

Comment:
This could be simplified, and avoid using type-casting:
```suggestion
  const data: VersionData[] = await response.json();
  const releases = data.map(({ name }) => name);
```

Reply:
Unfortunately the first cast is needed here as I can't find a way to tell `'isomorphic-unfetch'` what I expect the type of the json to be. The second cast isn't needed though, as by then the type of `"data"` can be inferred from the first cast, so I've removed this one